mkt/webapps/indexers.py

```diff
@@ -30,9 +49,10 @@ def search(cls, using=None):
 
         We override this to use our patched version which adds statsd timing.
         """
-        return Search(using=using or cls.get_es(),
-                      index=cls.get_index(),
-                      doc_type=cls.get_mapping_type_name())
+        return (Search(
+            using=using or cls.get_es(), index=cls.get_index(),
+            doc_type=cls.get_mapping_type_name())
+            .extra(_source={'exclude': cls.hidden_fields}))
```

Comment:
Is this the same change as the one in the parent class? Also, I don't see the statsd call the comment refers to?